extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -905,6 +908,22 @@ public void produceLoggingCategories(BuildProducer<LogCategoryBuildItem> categor
         }
     }
 
+    @BuildStep(onlyIf = NativeBuild.class)
+    public void test(CombinedIndexBuildItem index,
+            BuildProducer<ReflectiveClassBuildItem> reflective) {
+        Collection<AnnotationInstance> annotationInstance = index.getIndex().getAnnotations(STATIC_METAMODEL);
+        if (annotationInstance != null) {
+
+            String[] metamodel = annotationInstance.stream()
+                    .map(a -> a.target().asClass().toString())
```

Comment:
Please use `name().toString()` instead of the `ClassInfo#toString()`. In this case, it makes no difference but we had some issues with that before.

Reply:
Done